ocaml/xapi/db_gc.ml

```diff
@@ -20,6 +20,7 @@ open Stdext
 open Fun
 open Listext
 open Threadext
+open Db_gc_util
```

Comment:
I open it here because of the affinity of db_gc and util modules.
However you are right as well, specify module name before function will enhance readability.